.gitignore

```diff
@@ -6,3 +6,4 @@ dist/
 unversioned/
 cabal.sandbox.config*
 .cabal-sandbox
+TAGS
```

Comment:
Before, there was a trailing newline on the file.  This is a really minor point, but generally we have newlines at the end of the file.  My emacs adds this on save, so not having a newline at the end of a file can be annoying because saving the file adds in that extra change.